pkg/operator/ceph/object/controller.go

```diff
@@ -253,8 +253,14 @@ func (r *ReconcileCephObjectStore) reconcile(request reconcile.Request) (reconci
 		logger.Debugf("deleting store %q", cephObjectStore.Name)
 
 		if ok {
-			// Close the channel to stop the healthcheck of the endpoint
-			close(r.objectStoreChannels[cephObjectStore.Name].stopChan)
+			select {
+			case <-r.objectStoreChannels[cephObjectStore.Name].stopChan:
+				// channel was closed
```

Comment:
This is checking that something else closed the channel? Why was it closed multiple times?

Reply:
I don't know somehow I got closed multiple time, I know this shouldn't be happening, but this safeguard makes the operator not crash so it's important to have it.